server/db/init.js

```diff
@@ -22,7 +23,53 @@ bookshelf.knex.schema.hasTable('users').then((exist) => {
       table.timestamp('date_created').defaultTo(bookshelf.knex.fn.now());
       table.timestamp('date_updated').defaultTo(bookshelf.knex.fn.now());
     }).then(() => {
-      console.log(('Created users table'));
+      console.log('Created users table');
     });
   }
 });
+
+bookshelf.knex.schema.hasTable('scripts').then((exist) => {
+  if (!exist) {
+    bookshelf.knex.schema.createTable('scripts', (table) => {
+      table.increments('id').primary();
+      table.string('name').notNullable().index();
+      table.text('body').notNullable();
+      table.text('description').notNullable();
+      table.timestamp('created_at').defaultTo(bookshelf.knex.fn.now());
+      table.timestamp('updated_at').defaultTo(bookshelf.knex.fn.now());
+    }).then(() => {
+      console.log('Created scripts table');
+    }).catch(err => console.log('Error creating scripts table', err));
+  }
+});
+
+bookshelf.knex.schema.hasTable('questions').then((exist) => {
+  if (!exist) {
+    bookshelf.knex.schema.createTable('questions', (table) => {
+      table.increments('id').primary();
+      table.string('title').notNullable().index();
+      table.text('description').notNullable().index();
+      table.enu('type', ['multiselect', 'singleselect', 'paragraph']).notNullable();
+      table.text('responses').notNullable();
+      table.timestamp('created_at').defaultTo(bookshelf.knex.fn.now());
+      table.timestamp('updated_at').defaultTo(bookshelf.knex.fn.now());
+    }).then(() => {
+      console.log('Created questions table');
+    }).catch(err => console.log('Error creating questions table', err));
+  }
+});
+
+bookshelf.knex.schema.hasTable('questions_scripts').then((exist) => {
+  if (!exist) {
+    bookshelf.knex.schema.createTable('questions_scripts', (table) => {
```

Comment:
Maybe add a unique constraint for script_id and question_id?
`table.unique(['script_id', 'question_id])`